code/code/misc/discord.h

```diff
@@ -1,19 +1,29 @@
 #pragma once
 
 #include "sstring.h"
+#include <queue>
+#include <thread>
+#include <mutex>
+#include <utility>
 
 class Discord {
-    private: 
-        Discord();
-        static bool sendMessageAsync(sstring channel, sstring msg);
+  private:
+    Discord();
```

Comment:
Unimplemented private constructors are so 2010. Since C++11, we have `Discord() = delete`;

Reply:
Done